assembly/src/ast/nodes/serde/mod.rs

```diff
@@ -1,10 +1,9 @@
-use super::{
-    ByteReader, ByteWriter, CodeBody, Deserializable, DeserializationError, Felt, Instruction,
-    Node, ProcedureId, RpoDigest, Serializable,
-};
+use super::{CodeBody, Felt, Instruction, Node, ProcedureId, RpoDigest};
 use crate::MAX_PUSH_INPUTS;
 use num_enum::TryFromPrimitive;
 
+use vm_core::utils::{ByteReader, ByteWriter, Deserializable, DeserializationError, Serializable};
```

Comment:
Maybe we should consider standardising the grouping of imports across our crates.  I believe `rustfmt` has an option for this (but I don't think it's stable) - https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports.

Reply:
I tried this at some point and it looked a bit "messy" to me. But not a strong preference and if others like it, I'm happy to revisit.